tools/vagrant/README.md

```diff
@@ -75,7 +75,17 @@ wsk -i action invoke /whisk.system/utils/echo -p message hello --blocking --resu
     "message": "hello"
 }
 ```
-**Tip:** To connect to a different host API (i.e. bluemix.net) with the CLI, you will need to 
+
+**Tip:** You need to use the `-i` switch as the default SSL certificate used by the Vagrant installation is self-signed. Alternatively, you can configure your __apihost__ to use the non-SSL interface:
+
+```
+wsk -i property set --apihost http://192.168.33.13:10001 --auth `vagrant ssh -- cat openwhisk/ansible/files/auth.guest
```

Comment:
> Also isn't the whole point of this command to NOT use the `-i` and it's the line.

That's a good question. I assumed it was required as we haven't told it to use a non-secure host. However, thinking about it, I'm guessing that `wsk property set` works locally when setting the apihost so the `-i` is immaterial?